setup.py

```diff
@@ -96,7 +96,8 @@ def cmake_generate():
         '-DPYTHON_EXECUTABLE=' + sys.executable,
         '-DOTIO_PYTHON_INSTALL:BOOL=ON',
         '-DOTIO_CXX_INSTALL:BOOL=ON',
-        '-DCMAKE_BUILD_TYPE=' + ('Debug' if _ctx.debug else 'Release')
+        '-DCMAKE_BUILD_TYPE=' + ('Debug' if _ctx.debug else 'Release'),
+        '-DPYBIND11_FINDPYTHON=ON',  # Smart tool to find python's libs
     ]
```

Comment:
I'm glad you worked this out ~ I hadn't come across this yet. Would you mind adding a single comment here pointing us back to the documentation, just a link to  https://pybind11.readthedocs.io/en/latest/compiling.html#findpython-mode would be fine

Reply:
Done!